pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -254,13 +267,9 @@ func FindCandidates(ctx context.Context, cs kubernetes.Interface,
 		return nil, err
 	}
 
-	fco := &findCandidatesOpts{}
+	fco := &FindCandidatesOptions{}
```

Comment:
Note that if a caller doesn't put any `With` options, the values will be all zero, thus 0 candidates will be returned. That wouldn't be a good default. I think the default can be just 100%. @Huang-Wei ?

Reply:
True, however, a neat default configuration for default-scheduler outweighs the out-of-tree callers IMO.

How about adding a comment in FindCandidates() to remind callers to set the percentage to 100 if they want to return all candidates?

Reply:
I think it's cleaner to add a sanity check in `calculateNumCandidates` so that if the returned value is <= 0, set it to `numNodes`. Done.